.github/workflows/publish-gh-pages.yml

```diff
@@ -0,0 +1,123 @@
+name: generate-docs-pages
+
+on:
+  push:
+    branches:
+      - main
+
+concurrency: dbt_integration_tests
+
+env:
+  # Set profiles.yml directory
+  DBT_PROFILES_DIR: ./ci
+
+  # Redshift Connection
```

Comment:
Why not remove the credentials/steps that are unused instead of commenting out?

Reply:
Keeping for incase we ever need to change what we build from